### PR TITLE
chore: adjust CODEOWNERS

### PR DESCRIPTION
**Pull request by zepatrik**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*            @zepatrik
+
+/docs/docs/  @ory/documenters
```
Comment (aeneasr) [inline .github/CODEOWNERS:3]: I think we could even give `/docs` as they could merge also changes to the template plus versioned docs :)

**Review by aeneasr — COMMENTED** [review]
Nice! Could you also add this to all the other repos?

**Comment by aeneasr** [comment]
I mean just the big repos, not every one of the 150

File: .github/CODEOWNERS (hunk 1 above)
Comment (zepatrik) [reply .github/CODEOWNERS:3]: ah right forgot versioned docs

**Comment by zepatrik** [comment]
worked :tada: 
Screenshot from 2021-08-02 14-13-00

**Comment by zepatrik** [comment]
> I mean just the big repos, not every one of the 150

sure, on my way

**Pull request merged**